models/BuddiesModel.js

```diff
@@ -0,0 +1,223 @@
+import { MongoClient } from 'mongodb';
+import dotenv from 'dotenv';
+import {
+  checkUserCache,
+  addUserToCache,
+  userCache,
+} from '../src/utils/userCache.js';
+import { SIMPLE_MODELS } from './SIMPLE_MODELS.js';
+
+dotenv.config();
+
+const client = new MongoClient(process.env.MONGO_URI);
```

Comment:
why instantiate the db here instead of using inside the `BuddiesModel` on the construct as a prop?

Reply:
I don't think it needs to be part of the class, I think the connecting to differnet collections and such should absolutely be (which there are future plans for), but we can just create the client in this global scope and access it from the class.

That is my thinking on this, are there advantages you can think of for including it as part of the constructor? or am I missing anything?

Reply:
not as for right now. all that would do is put all the code be in the same place. but that's all.